### ISSUE TITLE
Is there a pytorch version of the code?

### ISSUE THREAD

**Issue by potoyeee** [Linfeng-Tang/DRLIE#1]
Is there a pytorch version of the code?

**Comment by Linfeng-Tang** [comment]
At present we have only implemented the proposed method based on TensorFlow. And we recommend you train a new model with Pytorch based on [DRIT](https://github.com/HsinYingLee/DRIT).

**Comment by zhuyr97** [comment]
> At present we have only implemented the proposed method based on TensorFlow. And we recommend you train a new model with Pytorch based on [DRIT](https://github.com/HsinYingLee/DRIT).

Will you release the corresponding PyTorch codes soon?

**Comment by Linfeng-Tang** [comment]
We don't have this plan at the moment

**Comment by 123Wynne** [comment]
您好，想请问一下，在运行main.py时，报错__init__() got multiple values for argument 'pool_size'，报错的地方在ops.py中第226行:return tf.compat.v1.keras.layers.AveragePooling2D(x, pool_size=kernel, strides=stride, padding='VALID')，请问该如何修改？
